src/engine/build.sh

```diff
@@ -247,12 +250,12 @@ case $B2_TOOLSET in
     ;;
 
     intel-linux)
-        CXX=${CXX:=icc}
+        CXX=${CXX:=icpc}
         CXX_VERSION_OPT=${CXX_VERSION_OPT:=--version}
         test_path ${CXX} >/dev/null 2>&1
         if test $? ; then
```

Comment:
Spotted one more bug in the previous implementation:
```suggestion
        if test_path ${CXX} ; then
```